druid/src/event.rs

```diff
@@ -274,6 +281,16 @@ pub enum LifeCycle {
     /// See [`is_hot`](struct.EventCtx.html#method.is_hot) for
     /// discussion about the hot status.
     HotChanged(bool),
+    /// This is called when the widget-tree changes and druid wants to rebuild the
+    /// Focus-chain.
+    ///
+    /// It is the only place from witch [`register_for_focus`] should be called.
+    /// By doing so the widget can get focused by other widgets using [`focus_next`] or [`focus_prev`].
+    ///
+    /// [`register_for_focus`](struct.LifecycleCtx.html#method.register_for_focus)
+    /// [`focus_next`](struct.LifecycleCtx.html#method.register_for_focus)
+    /// [`focus_prev`](struct.LifecycleCtx.html#method.register_for_focus)
```

Comment:
Let's use the new intra-doc links style:
```suggestion
    /// [`register_for_focus`]: crate::LifecycleCtx::register_for_focus
    /// [`focus_next`]: crate::EventCtx::focus_next
    /// [`focus_prev`]: crate::EventCtx::focus_prev
```